guides/common/modules/proc_editing-a-resource-quota.adoc

```diff
@@ -2,9 +2,35 @@
 = Editing a resource quota
 
 You can edit a resource quota to adjust available resources to {Project} users.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-editing-a-resource-quota[].
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Configure* > *Resource Quotas*.
 . Select your resource quota.
 . Adjust the *CPU cores*, *Memory*, or *Disk space* and click *Apply* to submit your changes to {Project}.
 . Click *Create resource quota* to submit the resource quota to {Project}.
+
+[id="cli-editing-a-resource-quota"]
+.CLI procedure
+. Optional: List all resource quotas:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer resource-quota list --fields id,name
+----
+. Edit a resource quota:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer resource-quota update \
```

Comment:
You can also user the `--description` option for the update command.

Reply:
Added.